source/fab/steps/preprocess.py

```diff
@@ -36,7 +37,7 @@ class MpCommonArgs(object):
 
 
 def pre_processor(config: BuildConfig, preprocessor: str,
-                  source_getter: ArtefactsGetter, output_collection, output_suffix,
+                  files: Collection[Path], output_collection, output_suffix,
```

Comment:
Why the change from "getter" to list of files? Do we not want to get these artefacts any more?

Reply:
Because, in the fortran preprocessor which wraps this, we want to get the list of files for the getter once, then filter it for big and little f90. This keeps the capability for the user to specify the getter themselves, or for us to provide a default, and for whichever it is to be called once.